Lesson04/HW04.SalaryBonusCalculator/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+
+namespace HW04.SalaryBonusCalculator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string basic;
+            double experience;
+            int experience1 = 5;
+            int experience2 = 10;
+            int experience3 = 15;
+            int experience4 = 20;
+            int experience5 = 25;
+            
+
+            Console.WriteLine("Введите вашу зарплату");
+            basic = Convert.ToString(Console.ReadLine());
+            Console.WriteLine("Ваша зарплата" + " " +basic);
+
+            Console.WriteLine("Введите ваш стаж работы");
+            experience = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Ваш стаж" + " " +experience);
```

Comment:
Everything is fine, but what if the user enters "asdnkdnkvasdasdsczxcasddasd"

Reply:
Done